### PR TITLE
Tweak media queries

### PR DESCRIPTION
**Pull request by randompearl-stack**

### CODE REVIEW

File: static/site.css [hunk 1]
```diff
@@ -428,11 +428,6 @@ iframe {
   display: grid;
   grid-gap: 2vmin;
 }
-@media (max-width: 799px) and (orientation: portrait) {
-  .lobby {
-    grid-template-areas: 'app' 'table' 'side' 'uleft' 'utable' 'blog';
-  }
-}
```
Comment (randompearl-stack) [inline static/site.css:435]: overriden by https://github.com/gbtami/pychess-variants/blob/0fc3e6b257af2da5da33b9b625067aec0b11544e/static/lobby.css#L10-L13

File: static/site.css [hunk 2]
```diff
@@ -767,7 +762,7 @@ ul.guide li {
   justify-content: flex-start;
   padding-left: 14px;
 }
-@media (max-width: 799px) and (orientation: portrait) {
+@media (max-width: 799px) {
```
Comment (randompearl-stack) [inline static/site.css:765]: prevents a bug where when `height < width < 799` the hamburger menu does not work

File: static/site.css [hunk 3]
```diff
@@ -1579,12 +1574,6 @@ table.seeks {
   font-weight: inherit;
   padding: 0.5em 0.7em 0.5em 0.7em;
 }
-@media (max-width: 799px) and (orientation: portrait) {
-  .seeks th,
-  .seeks td {
-    font-weight: 300;
-  }
-}
```
Comment (randompearl-stack) [inline static/site.css:1587]: There is no difference in appearance

File: static/site.css (hunk 2 above)
Comment (randompearl-stack) [reply static/site.css:765]: https://github.com/gbtami/pychess-variants/assets/126312812/72696ff1-bb0e-451c-8609-ac7b8e3cf7f7

**Comment by gbtami** [comment]
Thx.

**Pull request merged**